xen/arch/arm/domain_build.c

```diff
@@ -1590,6 +1590,20 @@ int __init make_chosen_node(const struct kernel_info *kinfo)
     return res;
 }
 
+#ifdef CONFIG_SCMI_SMC
+static int __init mem_permit_access(struct domain *d, uint64_t addr, uint64_t len)
+{
+    int res;
+    res = iomem_permit_access(d, paddr_to_pfn(addr),
+            paddr_to_pfn(PAGE_ALIGN(addr + len - 1)));
+    if ( res )
+        return res;
+
+    return map_regions_p2mt(d, gaddr_to_gfn(addr), PFN_DOWN(len),
```

Comment:
Should you remove iomem access permission if this call is failed?

Reply:
I don't see where it was fixed.

Reply:
Merged these changes to wrong commit. I'm sorry... now fixec